spec/std/openssl/ssl/server_spec.cr

```diff
@@ -67,8 +67,6 @@ describe OpenSSL::SSL::Server do
           client.close
         end
 
-        Fiber.yield
```

Comment:
For this spec, how can we be sure that we actually stressed the path we want?

That happened to me while testing crystal-db and I made a shameless helper `with_witness` that will fail if the `#check` is not called within the block. Because when the `.should` are inside the block, it might never get called.

```crystal
it "..." do
  with_witness do |w|
    spawn do
      some.should eq "..."
      w.check
    end
    # do something else, but we know the spawn 
    # is called before exiting the spec if we don't 
    # wrap with a with_witness?
  end
end
```

Reply:
By sending a response and reading it?

Reply:
But all the code happen in blocks. This spec is `nop` if open don't yield to the block.
Before with the `Fiber.yield` it seems that there was an explicit path that was intended.

I know the flaw was present before actually. But wont such help be an idiom worth to use for this an maybe other?

Reply:
Probably, but in this case expecting a response was simpler, and it enhances the spec a little —we can communicate both ways.